src/cmake/utils.cmake

```diff
@@ -68,15 +70,16 @@ function(add_library_copy_target)
 
     add_custom_target(
         ${PARSED_ARGS_TARGET}_copy_lib
-        DEPENDS ${PARSED_ARGS_TARGET} ${PARSED_ARGS_TARGET}_copy_lib_command
+        DEPENDS ${PARSED_ARGS_TARGET} ${PARSED_ARGS_TARGET}_copy_lib_command_64
```

Comment:
I think we don't need `_64` suffix here, as well as in `NEO_OUTPUT_LIBRARY_PATH_64`.